org.gnome.Ptyxis.json

```diff
@@ -0,0 +1,97 @@
+{
+    "app-id" : "org.gnome.Ptyxis",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "46beta",
```

Comment:
there is no 46beta on flathub, so this will likly have to wait a bit longer to build.

Reply:
Is it possible to have it as a `org.gnome.Ptyxis//beta` until `org.gnome.Platform//46` is out?

Reply:
we could merge it for beta, yes.

Reply:
If you want to test a build, 46 is out on Flathub